src/content/enterprise/administration/volume-snapshots.mdx

```diff
@@ -309,4 +310,6 @@ spec:
 </TabItem>
 </Tabs>
 
-As part of the creation of the persistent volume claim, Okteto will import the specified snapshot into your cluster using a `VolumeSnapshotContent` and will set the source of your persistent volume claim to this [VolumeSnapshotContent](https://kubernetes.io/docs/concepts/storage/volume-snapshots/), ready to be used by your development environment.
\ No newline at end of file
+As part of the creation of the persistent volume claim, Okteto will import the specified snapshot into your cluster using a `VolumeSnapshotContent` and will set the source of your persistent volume claim to this [VolumeSnapshotContent](https://kubernetes.io/docs/concepts/storage/volume-snapshots/), ready to be used by your development environment.
+
+Allowing volume snapshots from a snapshot-id is enabled by default, you can disable this feature by changing the configuration `allowIDAnnotation` to false in case you want to prevent PVCs created from any snapshot-id at the cluster.
```

Comment:
```suggestion
By default, anyone can use any of your cloud provider's snapshot ID as a data source. You can disable this by changing the configuration `allowIDAnnotation ` to `false` in the `volumeSnapshots` section of the configuration.
```

Reply:
is `data source` the right term?

Reply:
i guess it is, as when using the native k8s key to provide this at the pvc spec is dataSource